src/flow_solver/flow_solver.cpp

```diff
@@ -475,17 +480,47 @@ ::create_FlowSolver(const Parameters::AllParameters *const parameters_input,
     return nullptr;
 }
 
+template<int dim, int nstate>
+std::unique_ptr< FlowSolverBase > FlowSolverFactory<dim,nstate>
```

Comment:
FlowSolverFactory could be moved to a new file to match other factories in the code like for ODE and Physics. The FlowSolver class is getting big, so maybe its time they each get their own file.